experiment/scheduler.py

```diff
@@ -585,23 +586,28 @@ def schedule_loop(experiment_config: dict):
     # Create the thread pool once and reuse it to avoid leaking threads and
     # other issues.
     logger.info('Starting scheduler.')
-    gce.initialize()
     num_trials = len(
         get_experiment_trials(experiment_config['experiment']).all())
-    trial_instance_manager = TrialInstanceManager(num_trials, experiment_config)
+    local_experiment = experiment_utils.is_local_experiment()
+    if not local_experiment:
+        gce.initialize()
+        trial_instance_manager = TrialInstanceManager(num_trials,
+                                                      experiment_config)
     experiment = experiment_config['experiment']
     with multiprocessing.Pool() as pool:
         handle_preempted = False
         while not all_trials_ended(experiment):
             try:
-                if not handle_preempted and not any_pending_trials(experiment):
-                    # Only start handling preempted instances once every initial
-                    # trial was started. This ensures that .
-                    handle_preempted = True
-
                 schedule(experiment_config, pool)
-                if handle_preempted:
-                    trial_instance_manager.handle_preempted_trials()
+                if not local_experiment:
```

Comment:
I didn't quite mean this, sorry for being unclear.
Let's clean it up a bit. Let's change this so that handle_preempted never becomes True when running locally

Reply:
Got the idea. Let me fix it tonight.

Reply:
Fixed in #464